app/src/main/java/org/mozilla/fenix/library/history/HistoryController.kt

```diff
@@ -44,7 +45,7 @@ class DefaultHistoryController(
         when (item) {
             is History.Regular -> openToBrowser(item)
             is History.Group -> {
-                metrics.track(Event.HistorySearchTermGroupTapped)
+                org.mozilla.fenix.GleanMetrics.History.searchTermGroupTapped.record(NoExtras())
```

Comment:
Is there a reason we're not `import org.mozilla.fenix.GleanMetrics.History`?  This applies to other cases as well.